crypto/rsa/rsa_backend.c

```diff
@@ -229,6 +229,12 @@ int ossl_rsa_fromdata(RSA *rsa, const OSSL_PARAM params[], int include_private)
         }
     }
 
+    if (!ossl_rsa_check_factors(rsa)) {
+        ERR_raise_data(ERR_LIB_RSA, ERR_R_INTERNAL_ERROR,
```

Comment:
RSA_R_INVALID_KEYPAIR or ERR_R_PASSED_INVALID_ARGUMENT would be better error codes